.haml-lint_todo.yml

```diff
@@ -13,7 +13,7 @@ linters:
 
   # Offense count: 189
   SpaceInsideHashAttributes:
-    enabled: false
+    enabled: true
```

Comment:
Werent we just going to delete these as we completed them?

Reply:
My bad - deleted it before the add! will push up fix now